utils/details.js

```diff
@@ -369,6 +374,9 @@ const searchQuery = `query ($search: String!) {
             type: __typename
             login
             url
+            ... on Organization {
+              isVerified
+            }
```

Comment:
@digimangos, this is to verify 
> [Whether the organization has verified its profile email and website.](https://docs.github.com/en/graphql/reference/objects#organization)

Is that enough, or are we looking to add if the marketplace listing `isVerified`? Because then one would need to query https://docs.github.com/en/graphql/reference/queries#marketplacelisting instead.

Reply:
I’ll look in to that, I think the more meta data we can retrieve to help assess the risk profile the better.

Reply:
Hmmm @stoe , not sure at the moment how I can retrieve the listing. If I take a random example from the marketplace. The details are: `oak9io/oak9.githubaction@v1.0.3`, this relates to a market place slug of: `oak9-github-action`. But there is no way for me to know that. I would need to somehow search the marketplace to look for a listing which relates to the `owner/repo` for the action specified unless I’m missing something.

Reply:
@digimangos, we could retrieve the slug from the `action.yml` in 
https://github.com/ActionsDesk/github-actions-details-action/blob/31132c502fb64ffa98f72a7e1f4dc2fcc3282856/utils/details.js#L154-L158

It's the `name` property with spaces replaced by `-` and all lowercase.

I did test it, but looks like https://docs.github.com/en/graphql/reference/queries#marketplacelisting only retrieves GitHub Apps listed, not Actions.

Reply:
That said, should we indicate that the verification information is only about the organization and not about the Marketplace badge?

Reply:
I updated the description at https://github.com/ActionsDesk/github-actions-details-action/blob/80fbd65717ad6f0d9e17650dd02e8b86da9c8784/utils/details.js#L265 to `Organization verified`